src/chocolatey/infrastructure.app/commands/ChocolateyPackCommand.cs

```diff
@@ -104,5 +108,25 @@ public virtual bool may_require_admin_access()
         {
             return false;
         }
+
+        private void parse_properties(string input, IDictionary<string, string> properties)
+        {
+            input = input.remove_surrounding_quotes();
+            if (string.IsNullOrWhiteSpace(input)) return;
+
+            var tokens = input.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries);
+            foreach (var token in tokens)
+            {
+                var pair = token.Split(new[] { '=' }, 2, StringSplitOptions.RemoveEmptyEntries);
+                if (pair.Length != 2)
+                {
+                    throw new ArgumentException(
+                        string.Format("Ill-formed name=value pair in token: {0}", token),
+                        "properties");
+                }
+
+                properties.Add(pair[0], pair[1].remove_surrounding_quotes());
+            }
+        }

```

Comment:
Okay. Maybe I'll break that out into a common function. Could be an extension method itself in perhaps `StringExtensions`?